lighthouse-core/audits/dobetterweb/uses-passive-event-listeners.js

```diff
@@ -80,11 +80,21 @@ class PassiveEventsAudit extends Audit {
 
     const groupedResults = EventHelpers.groupCodeSnippetsByLocation(results);
 
+    const createTable = Formatter.getByName(
+        Formatter.SUPPORTED_FORMATS.TABLE).createTable;
+
+    const table = createTable({
+      url: 'URL', lineCol: 'Line/Col', type: 'Type', code: 'Snippet'
+    }, groupedResults);
+
     return PassiveEventsAudit.generateAuditResult({
       rawValue: groupedResults.length === 0,
       extendedInfo: {
-        formatter: Formatter.SUPPORTED_FORMATS.URLLIST,
-        value: groupedResults
+        formatter: Formatter.SUPPORTED_FORMATS.TABLE,
+        value: {
+          results: groupedResults,
```

Comment:
nope. The table formatter will pull from the`groupedResults` object as needed, based on the keys used in the first arg to `createTable({url: 'URL', lineCol: 'Line/Col', type: 'Type', code: 'Snippet'})`.